probe-dictionary/explorer.css

```diff
@@ -131,3 +131,8 @@ body {
 .nav-item {
     font-weight: bold;
 }
+th{
```

Comment:
The style in this file is to have a full line of whitespace between rulesets, and a space between the selector and `{`

Reply:
@chutten I have corrected it.